core/javaAction/init.py

```diff
@@ -18,12 +15,12 @@
     encoded = sys.argv[1]
 
 payload = {
```

Comment:
we may delete this script now (with this change https://github.com/openwhisk/openwhisk/pull/1938/files#diff-80a0161d4fae50b554066617d1de0239R83, i think the `invoke.py` script in `actionproxy` will work for all the runtimes although i did not verify).